modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-order-impl/src/main/java/com/liferay/headless/commerce/admin/order/internal/resource/v1_0/OrderResourceImpl.java

```diff
@@ -865,11 +865,15 @@ private CommerceOrder _updateOrder(CommerceOrder commerceOrder, Order order)
 					order.getPrintedNote(), commerceOrder.getPrintedNote()));
 		}
 
-		commerceOrder = _commercePaymentEngine.updateOrderPaymentStatus(
-			commerceOrder.getCommerceOrderId(),
-			GetterUtil.getInteger(
-				order.getPaymentStatus(), commerceOrder.getPaymentStatus()),
-			commerceOrder.getTransactionId(), StringPool.BLANK);
+		if ((order.getPaymentStatus() != null) &&
```

Comment:
@RenanUmV This change doesn't look like an SF, please separate it and explain why.